### PR TITLE
Add `contrib` directory

### LINKED ISSUE #1178: Add `contrib/` directory
[TBC

### PR DESCRIPTION
**Pull request by astrojuanlu**
As well as other minor documentation fixes I made along the way.

Closes #1178.

### CODE REVIEW

**Comment by astrojuanlu** [comment]
cc @sammmlow :)

**Comment by DhruvSondhi** [comment]
Looks Awesome 🚀

File: contrib/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+please follow these steps:
+
+- If it's only one script,
+  or comment at the top
+  specifying your name, email,
+  what the script does,
+  what extra libraries are needed (if any),
+  and cite any scientific references related to your work.
+- If it's a series of scripts,
+  create a new directory with a `lower_case` name
```
Comment (jorgepiloto) [inline contrib/README.md:23]: We might rewrite this as " create a lowercase named directory having a REDME.md file holding your name...".

**Review by jorgepiloto — COMMENTED** [review]
This looks good @astrojuanlu! I left just a couple of suggestions. In addition, I saw some unnecessary break of lines before reaching the 80 character column, not sure if you noticed.

**Comment by sammmlow** [comment]
Super cool! So contrib will be the white space to test out poliastro scripts. Nice idea! Forked and working on a relative astrodynamics module :)

If it is OK, perhaps I can also add onto the documentation on this

**Comment by astrojuanlu** [comment]
@sammmlow Awesome! About the documentation, as scripts in `contrib/` will not be part of the poliastro API, it's enough to describe in the corresponding `README.md` everything that needs to be explained.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <p align="center">
-    <a href="http://poliastro.github.io/"><image src="http://poliastro.github.io/images/logo_text.png" alt="poliastro logo" width="675px"></a>
+    <a href="http://www.poliastro.space/"><image src="https://www.poliastro.space/images/logo_text.png" alt="poliastro logo" width="675px"></a>
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply README.md:2]: For these small typos, you can use the "suggestions" feature of GitHub, which makes it very easy! See:

```suggestion
    <a href="https://www.poliastro.space/"><image src="https://www.poliastro.space/images/logo_text.png" alt="poliastro logo" width="675px"></a>
```

File: contrib/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Community contributions
+
+The `contrib` directory contains community contributions
+that are not yet ready to be part of the poliastro API,
+but are nonetheless useful for Astrodynamics Python users.
+
+If you have a small script or set of scripts
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply contrib/README.md:7]: Good one!

```suggestion
If you have a small script or set of them
```

File: contrib/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- If it's only one script,
+  or comment at the top
+  specifying your name, email,
+  what the script does,
+  what extra libraries are needed (if any),
+  and cite any scientific references related to your work.
+- If it's a series of scripts,
+  create a new directory with a `lower_case` name
+  with a `README.md` specifying your name, email,
```
Comment (astrojuanlu) [inline contrib/README.md:24]:
```suggestion
  create a new `lower_case` named directory
  having a `README.md` specifying your name, email,
```

**Comment by astrojuanlu** [comment]
Thanks for the comments @jorgepiloto ! About line breaks, I tried to apply this idea: https://rhodesmill.org/brandon/2012/one-sentence-per-line/ it looks "uglier" in plain text but it helps with version control I think, and the appearance is the same.

**Comment by jorgepiloto** [comment]
> I tried to apply this idea: https://rhodesmill.org/brandon/2012/one-sentence-per-line/ it looks "uglier" in plain text but it helps with version control I think, and the appearance is the same.

Ah! I was sure you applied some kind of reasoning behind those lines. That "semantic linefeeds" approach looks very interesting and seems to provide also fast reading/review of the text. Cool! 🚀

**Pull request merged**